manim/logger.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from rich.logging import RichHandler
+
+## Setting config for logger.
+logging.basicConfig(
+    level="NOTSET",
+    format="%(message)s",
```

Comment:
I actually think the filename could come in handy in logs. Maybe something that could be flag-configurable later.